Modules/System Test Libraries/Environment Information/src/EnvironmentInfoTestLibrary.Codeunit.al

```diff
@@ -19,30 +19,63 @@ codeunit 135094 "Environment Info Test Library"
     /// This functions should only be used for testing purposes.
     /// </remarks>
     /// <param name="EnableSandboxForTest">The value to be set to the testability sandbox flag.</param>
-    [Scope('OnPrem')]
```

Comment:
I believe it is fine to move these away from OnPrem scope as we do not publish the libraries in SaaS. Will need to include the security people to review.